playbooks/check_fips_status.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Check FIPS status
```

Comment:
should we just call this "FIPS status" everywhere without the "check"? The default mode is to check, but you can also use it to set. It seems weird to encode the default behavior in the name.

Reply:
Makes sense.